src/shared/sam/debugger/awsSamDebugConfiguration.ts

```diff
@@ -20,9 +20,9 @@ export * from './awsSamDebugConfiguration.gen'
 
 export const AWS_SAM_DEBUG_TYPE = 'aws-sam'
 export const DIRECT_INVOKE_TYPE = 'direct-invoke'
-export const TEMPLATE_TARGET_TYPE: 'template' = 'template'
-export const CODE_TARGET_TYPE: 'code' = 'code'
-export const API_TARGET_TYPE: 'api' = 'api'
+export const TEMPLATE_TARGET_TYPE = 'template' as const
```

Comment:
Is `as const` required after removing the type? This is being constructed as a const. If so, then do this for the other instances above and below this (and honestly everywhere else in the codebase)

Reply:
It's not needed. And you're right it's pretty redundant now that I'm looking at it again.

Reply:
Well looks like they actually were needed by the compiler.

Reply:
> If so, then do this for the other instances above and below this (and honestly everywhere else in the codebase)

-1. This is a special case to satisfy the compiler. And only because of the way we modeled `AwsSamDebugConfiguration`, which should be revisited.

Reply:
Out of curiosity, what's the compiler error when you drop the additional `as const`? Is the reason this flags and the others don't because we use `typeof` against this? Just making sure I understand.